CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Add `AsRef<[u8]>` for newtypes (#323)
 * Implement memory locking and unlocking (#324)
 * Add convenience functions for pwhash() and derive_key() (#309)
+* Update libsodium to 1.0.18 (#342)
```

Comment:
0.2.2 was released on May 16, 2019, so you have to create a new patch in changelog.

Reply:
Fixed.